versioned_docs/version-8.6/guides/getting-started-java-spring.md

```diff
@@ -50,7 +50,7 @@ If no version of Java is found, follow your chosen installation's instructions f
 Next, create a new Spring Boot project:
 
 1. Go to [https://start.spring.io/](https://start.spring.io/) to get started.
-2. Under **Project**, select **Maven**. Under **Language**, select **Java**. Under **Spring Boot**, select the latest non-SNAPSHOT version (currently 3.4.1).
+2. Under **Project**, select **Maven**. Under **Language**, select **Java**. Under **Spring Boot**, select `3.3.8`.
```

Comment:
🔧 As the patch level might change anytime, wdyt about using X (or something better) as placeholder?
```suggestion
2. Under **Project**, select **Maven**. Under **Language**, select **Java**. Under **Spring Boot**, select `3.3.X`.
```

Reply:
I got feedback from Camundi that the X wasn't descriptive enough 🤷‍♀️ we are looking at potentially automating this in the future, so it's always up-to-date, but I could argue for either one.